CONTRIBUTING.md

```diff
@@ -361,14 +361,16 @@ Example label comment:
 
 ## Becoming a Code Owner
 
-A Code Owner is responsible for a component within Collector Contrib, as indicated by the [CODEOWNERS file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS). That responsibility includes maintaining the component, responding to issues, and reviewing pull requests.
+A Code Owner is responsible for a component within Collector Contrib, as indicated by the [CODEOWNERS file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS). That responsibility includes maintaining the component, triaging and responding to issues, and reviewing pull requests.
 
 Sometimes a component may be in need of a new or additional Code Owner. A few reasons this situation may arise would be:
-- The component was never assigned a Code Owner.
+
+- The existing Code Owners are actively looking for more help.
```

Comment:
Is there a way for existing code owners to communicate this? You mentioned in the sig possibly labelling components (IIRC)? Should there be some value that can be specified in `mdatagen.yaml` that could then be added to the component's README to communicate this?

Sorry, I see you're just moving this comment so this isn't meant to block your change, more of a question/discussion.

Reply:
That's the idea. I proposed something detailed in #31115